packages/Webkul/Velocity/src/Resources/views/admin/content/edit.blade.php

```diff
@@ -8,6 +8,7 @@
     <div class="content">
         @php
             $locale = request()->get('locale') ?: app()->getLocale();
+            $translations = $content->translations->where('locale', $locale)->first();
```

Comment:
Change the variable name to $translation because it contains a single translation.

Reply:
Variable Name Changed